src/main/java/io/passport/server/controller/DatasetTransformationStepController.java

```diff
@@ -149,12 +194,26 @@ public ResponseEntity<?> deleteDatasetTransformationStep(@PathVariable Long step
                 return ResponseEntity.status(HttpStatus.FORBIDDEN).build();
             }
 
-            boolean isDeleted = this.datasetTransformationStepService.deleteDatasetTransformationStep(stepId);
-            return isDeleted ? ResponseEntity.noContent().build() : ResponseEntity.notFound().build();
+            Optional<DatasetTransformationStep> deletedDatasetTransformationStep = this.datasetTransformationStepService.deleteDatasetTransformationStep(stepId);
+            if (deletedDatasetTransformationStep.isPresent()) {
+                String description = Description.DELETION.getDescription(relationName, stepId.toString());
```

Comment:
I see that we pass Operation.DELETE, relationName and stepId.toString to createAuditLog function. Therefore, I wonder whether we can directly create the description inside this function without passing it directly?